src/components/manage/Widgets/FileWidget.jsx

```diff
@@ -106,7 +105,11 @@ const FileWidget = (props) => {
                 className="image-preview"
                 id={`field-${id}-image`}
                 size="small"
-                src={value?.download ? flattenToAppURL(value.download) : null}
+                src={
```

Comment:
I have added params so that browser doesn't cache the image. It is very misleading when you keep changing the image for the right fit and the browser doesn't update the preview image.